packages/salesforcedx-sobjects-faux-generator/package.json

```diff
@@ -30,7 +30,7 @@
     "nyc": "^11.0.2",
     "sinon": "^2.3.6",
     "typescript": "2.4.0",
-    "vscode": "1.1.4"
+    "vscode": "1.1.5"
```

Comment:
Any reason for this bump? I do not oppose the bump but why are you bumping it only for yours?

Reply:
I should take it back out - it was when I was trying to use vscode - probably shouldn't depend on vscode at all for this non-vsix package.     I was just trying to make it consistent with the other ones -but best to remove it - I'll verify